### PR TITLE
Add version note for Remix manual Express wrapper.

### PR DESCRIPTION
**Pull request by onurtemizkan**
Adding a version note as per review: https://github.com/getsentry/sentry-docs/pull/5425#discussion_r946789444

### CODE REVIEW

File: src/includes/getting-started-config/javascript.remix.mdx [hunk 1]
```diff
@@ -44,6 +44,12 @@ Learn more about <Link to="/platforms/node/performance/database/opt-in/#prisma-o
 
 If you use a custom Express server in your Remix application, you should wrap your [`createRequestHandler` function](https://remix.run/docs/en/v1/other-api/adapter#createrequesthandler) manually with `wrapExpressCreateRequestHandler`. This is not required if you use the built-in Remix App Server.
 
+<Alert level="info">
+  
+`wrapExpressCreateRequestHandler` is available from version 7.11.0.
```
Comment (imatwawana) [inline src/includes/getting-started-config/javascript.remix.mdx:49]:
```suggestion
`wrapExpressCreateRequestHandler` is available starting with version 7.11.0.
```

**Review by imatwawana — APPROVED** [review]
One small thing, but otherwise looks good!

**Comment by imatwawana** [comment]
Shall I merge once checks are complete?

**Comment by onurtemizkan** [comment]
That would be great. Thanks!

**Pull request merged**